fgrpc/grpcrunner.go

```diff
@@ -51,6 +55,14 @@ func Dial(o *GRPCRunnerOptions) (conn *grpc.ClientConn, err error) {
 	case strings.HasPrefix(o.Destination, fnet.PrefixHTTPS):
 		creds := credentials.NewTLS(&tls.Config{InsecureSkipVerify: o.Insecure}) // nolint: gosec // explicit flag
 		opts = append(opts, grpc.WithTransportCredentials(creds))
+	case strings.HasPrefix(o.Destination, fnet.PrefixXDS):
+		// when using the xds:/// scheme, use
+		creds, err := xds.NewClientCredentials(xds.ClientOptions{FallbackCreds: insecure.NewCredentials()})
```

Comment:
why is insecure hardcoded vs using o.Insecure?

Reply:
I guess maybe because of "Fallback" what is fallback for, is it required?

Reply:
FallbackCreds says: "if the XDS server doesn't send me security configuration, fallback to this mode". This basically gives full control of security options to the control plane. I could require the `o.Insecure` to allow the fallback?